### ISSUE TITLE
Paddle 2.0.1 to Paddle 2.0.2 oneDNN inference performance drop

### ISSUE THREAD

**Issue by lidanqing-vv** [PaddlePaddle/Paddle#32311]
Some models in continuous_integrations Test performance drop due to oneDNN upgraded to oneDNN v2.2
DPN68 -8.51%,
MobilenetV2 -6.47%, 
ssdlite_mobilenet_v3_large -14.06%

**Comment by lidanqing-vv** [comment]
**DPN68 fp32: bs 1, thread 1**
**Release 2.0.1**
```
------------------------->     Profiling Report     <-------------------------

Place: CPU
Time unit: ms
Sorted by total time in descending order in the same thread

-------------------------     Overhead Summary      -------------------------

Total time: 29658.6
  Computation time       Total: 27588.4     Ratio: 93.0196%
  Framework overhead     Total: 2070.29     Ratio: 6.9804%

-------------------------     GpuMemCpy Summary     -------------------------

GpuMemcpy                Calls: 0           Total: 0           Ratio: 0%

-------------------------       Event Summary       -------------------------

Event                           Calls       Total       Min.        Max.        Ave.        Ratio.
thread0::conv2d                 36210       19890.8     0.138874    5.31951     0.549319    0.670659
thread0::batch_norm             13770       5714.62     0.097039    2.19846     0.415005    0.19268
thread0::concat                 22440       1708.25     0.021528    0.893382    0.0761254   0.0575972
thread0::split                  13260       1221.06     0.039974    2.52828     0.0920859   0.0411704
thread0::elementwise_add        11220       804.539     0.032675    0.326106    0.0717058   0.0271266
thread0::fc                     510         174.411     0.285406    25.0373     0.341983    0.00588062
thread0::pool2d                 1020        102.919     0.068084    0.994931    0.100901    0.00347013
thread0::load_combine           1           21.9712     21.9712     21.9712     21.9712     0.000740802
thread0::softmax                510         13.0211     0.02178     0.347654    0.0255316   0.000439033
thread0::scale                  510         7.01723     0.011612    0.057425    0.0137593   0.0002366
```


**Release 2.0.2**
```
------------------------->     Profiling Report     <-------------------------

Place: CPU
Time unit: ms
Sorted by total time in descending order in the same thread

-------------------------     Overhead Summary      -------------------------

Total time: 31843.6
  Computation time       Total: 29324.1     Ratio: 92.0878%
  Framework overhead     Total: 2519.53     Ratio: 7.91219%

-------------------------     GpuMemCpy Summary     -------------------------

GpuMemcpy                Calls: 0           Total: 0           Ratio: 0%

-------------------------       Event Summary       -------------------------

Event                           Calls       Total       Min.        Max.        Ave.        Ratio.
thread0::conv2d                 36210       21490.2     0.124802    29.766      0.593488    0.674867
thread0::batch_norm             13770       5933.96     0.178744    1.64772     0.430934    0.186347
thread0::concat                 22440       1682.62     0.020963    0.867991    0.074983    0.0528401
thread0::split                  13260       1675.41     0.051464    3.02778     0.126351    0.0526138
thread0::elementwise_add        11220       795.16      0.033637    0.334223    0.0708699   0.0249708
thread0::fc                     510         175.291     0.2864      25.6616     0.343707    0.00550474
thread0::pool2d                 1020        47.3454     0.037359    0.750164    0.0464171   0.00148681
thread0::load_combine           1           23.0921     23.0921     23.0921     23.0921     0.000725172
thread0::softmax                510         13.4579     0.02312     0.379566    0.026388    0.000422625
thread0::scale                  510         7.06579     0.011848    0.056944    0.0138545   0.00022189
```

**Comment by lidanqing-vv** [comment]
Close this issue since most models perf are better, and two models per are being fixed using oneDNN v2.3. Other two models are being fixed by adding "split" support
